docs/user-guide/templates.md

```diff
@@ -28,6 +28,21 @@ jobs:
           - NPM_TOKEN
 ```
 
+### Wrap
+Wrapping is when you add commands to run before and/or after an existing step. To wrap a step from a template, add a `pre` or `post` in front of the step name.
+
+Example:
+```yaml
+jobs:
+   main:
+      template: template_name@1.3.0
+      steps:
+          - preinstall: echo pre-install
+          - postinstall: echo post-install
+```
+
+This will run the command `echo pre-install` before the `install` step, and `echo post-install` after the `install` step.
```

Comment:
maybe:
before the template's `install` step
and
after the template's `install` step

And/or should we include an example of the combined screwdriver yaml?
```yaml

jobs:
    main:
        image: node:6
        steps:
            - preinstall: echo pre-install
            - install: npm install
            - postinstall: echo post-install
            - test: npm test
            - echo: echo $FOO
        environment:
            FOO: bar
        secrets:
            - NPM_TOKEN


```